### PR TITLE
24/-api]-change-user-group-+-update-scheme

### PR DESCRIPTION
**Pull request by CoachBenedetto**
I have updatet the graphql scheme in web-app as well as app (I had to move backend back to the frontend section, otherwise it wouldn't work). Futhermore I had to resetup the backend and reintegrate it to change the user group.

### CODE REVIEW

**Comment by CoachBenedetto** [comment]
Der autogen ordner in flotter lässt sich leider nicht verändern. Dementsprechend passt der commit so. Wenn du approvest kann ich morgen @apolat2000

**Comment by CoachBenedetto** [comment]
@apolat2000 habe jetzt die Admin Queries rausgeschmissen. Kannst du dann approven?

**Review by apolat2000 — APPROVED** [review]
LGTM :+1:

File: admin-app/frontend/amplify/backend/api/monitoring/schema.graphql [hunk 1]
```diff
@@ -56,7 +56,7 @@ type StoragePaths {
 type Level @model {
   name: String!
   description: String @default(value: "")
-  parentLevel: Level @hasOne
+  parentLevelID: ID
```
[the remark this answers is not in the document]
Comment (CoachBenedetto) [reply admin-app/frontend/amplify/backend/api/monitoring/schema.graphql:59]: No, only where circular dependencies (model referencing the same model) occur as amplify flutter cannot handle those when applied via connections.

File: mobile-app/amplify/backend/api/monitoring/schema.graphql [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+type Config @model {
+  name: String!
+  colorTheme: ColorTheme
+  storagePaths: StoragePaths!
+  schemeVersion: Int @default(value: "0")
+}
+
+type ColorTheme {
+  highlight: String # chosen by user # todo: default
+  secundaryHighlight: String # chosen by user # todo: default
```
[the remark this answers is not in the document]
Comment (CoachBenedetto) [reply mobile-app/amplify/backend/api/monitoring/schema.graphql:41]: Ups. Changed those

**Review by apolat2000 — APPROVED** [review]

**Pull request merged**